cumulusci/core/config/project_config.py

```diff
@@ -304,16 +315,8 @@ def repo_owner(self):
         if not self.repo_root:
             return
 
-        in_remote_origin = False
-        with open(os.path.join(self.repo_root, ".git", "config"), "r") as f:
-            for line in f:
-                line = line.strip()
-                if line == '[remote "origin"]':
-                    in_remote_origin = True
-                    continue
-                if in_remote_origin and line.find("url =") != -1:
-                    line_parts = line.split("/")
-                    return line_parts[-2].split(":")[-1]
```

Comment:
this is what prompted me to change the `_split_repo_url()` method to return the actual 'owner' segment of the url